Menu/ArenaLobbyMenu2.cs

```diff
@@ -62,8 +64,6 @@ public ArenaLobbyMenu2(ProcessManager manager) : base(manager, RainMeadow.Ext_Pr
         pages.Add(slugcatSelectPage = new Page(this, null, "slugcat select", 1));
         slugcatSelectPage.pos.x += 1500f;
 
-        ChangeScene(slugcatScene = Arena.slugcatSelectMenuScenes[Arena.arenaClientSettings.playingAs.value]);
-
         competitiveShadow = new(this, scene, "", "CompetitiveShadow", new Vector2(-2.99f, 265.01f), true, false);
```

Comment:
```suggestion
        ChangeScene(slugcatScene = Arena.slugcatSelectMenuScenes[Arena.arenaClientSettings.playingAs.value]);
```
This prevents an initial fade in for the menu bg.

Reply:
This apparently only prevents the initial fade-in if the slugcat is Survivor. Might want to take a look later